tests/system_test.py

```diff
@@ -18,39 +19,20 @@
 from fixedpointmath import FixedPoint
 from sqlalchemy.orm import Session
 
-# This pass is to prevent auto reordering imports from reordering the imports below
-pass  # pylint: disable=unnecessary-pass
-
-# Test fixture imports
-# Ignoring unused import warning, fixtures are used through variable name
-from agent0.test_fixtures import (  # pylint: disable=unused-import, ungrouped-imports
-    AgentDoneException,
-    cycle_trade_policy,
-)
-from chainsync.test_fixtures import db_session  # pylint: disable=unused-import, ungrouped-imports
-from ethpy.test_fixtures import local_chain, local_hyperdrive_chain  # pylint: disable=unused-import, ungrouped-imports
-
-# fixture arguments in test function have to be the same as the fixture name
-# pylint: disable=redefined-outer-name
+# These tests are using fixtures defined in conftest.py
 
 
 class TestLocalChain:
     """Tests bringing up local chain"""
 
-    # This is using 2 fixtures. Since hyperdrive_contract_address depends on local_chain, we need both here
-    # This is due to adding test fixtures through imports
-    def test_hyperdrive_init_and_deploy(self, local_chain: str, local_hyperdrive_chain: dict):
+    def test_hyperdrive_init_and_deploy(self, local_hyperdrive_chain: dict):
         """Create and entry"""
-        print(local_chain)
+        print(local_hyperdrive_chain["rpc_url"])
         print(local_hyperdrive_chain)
 
 
-def _to_unscaled_decimal(scaled_value: int) -> Decimal:
-    return Decimal(str(FixedPoint(scaled_value=scaled_value)))
-
-
-def _decimal_almost_equal(a_val: Decimal, b_val: Decimal) -> bool:
-    return abs(a_val - b_val) < 1e-12
+def _to_unscaled_decimal(fp_val: FixedPoint) -> Decimal:
```

Comment:
Instead can you add a method to FixedPoint to allow it to be a Decimal?
```python
def __decimal__(self) -> Decimal:
    r"""Cast to Decimal"""
    return Decimal(str(self))
```

Reply:
Are we still trying to maintain FixedPoint? I thought we were deprecating it in favor of rust